Content.Server/Lathe/LatheSystem.cs

```diff
@@ -98,13 +99,18 @@ private void OnComponentInit(EntityUid uid, LatheComponent component, ComponentI
 
         /// <summary>
         /// When someone tries to use an item on the lathe,
-        /// insert it if it's a stack and fits inside
+        /// insert it if it's a stack and fits inside and don't have anything stuck to it
         /// </summary>
-        private void OnInteractUsing(EntityUid uid, LatheComponent component, InteractUsingEvent args)
+        private void OnInteractUsingEvent(EntityUid uid, LatheComponent component, InteractUsingEvent args)
```

Comment:
why was the name changed? the previous one followed convention just fine

Reply:
Yeah I will change it back